### PR TITLE
Standardize the capitalization and short form of Red Hat Customer Portal

### PR DESCRIPTION
**Pull request by rolfedh**
Purpose: This PR standardizes the capitalization and short form of Red Hat Customer Portal.
Fixes https://github.com/redhat-documentation/supplementary-style-guide/issues/384

Request for Review:
@IngridT1, @mportman12, @bergerhoffer, @ndeevy, @bredamc - your insights and feedback on these changes would be greatly appreciated. Thank you.

### CODE REVIEW

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc [hunk 1]
```diff
@@ -286,12 +286,12 @@ The Red Hat Container Catalog no longer exists; it has become part of the Red Ha
 // EAP: General; kept as is
 [[red-hat-customer-portal]]
 ==== image:images/yes.png[yes] Red Hat Customer Portal (noun)
-*Description*: _Red Hat Customer Portal_ is the official name of the customer portal at https://access.redhat.com.
+*Description*: _Red Hat Customer Portal_ is the official name for https://access.redhat.com. Always use _Red Hat Customer Portal_ in the first reference. For subsequent mentions, you can shorten it to _Customer Portal_.
 
-*Use it*: yes
+*Use it*: Yes
```
Comment (bredamc) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:291]:
```suggestion
*Use it*: yes
```
All other glossary entries use lowercase "yes". If we want to make this change (to use the initial capital letter), we should make it consistently throughout the glossary.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc [hunk 2]
```diff
@@ -286,12 +286,12 @@ The Red Hat Container Catalog no longer exists; it has become part of the Red Ha
...
 
 [.vale-ignore]
-*Incorrect forms*: Customer Portal
+*Incorrect forms*: portal
```
Comment (bredamc) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:294]:
```suggestion
*Incorrect forms*: CP, RHCP
```

According to the OPL, "portal" is an acceptable short form, but "CP" and "RHCP" are not.

**Review by bredamc — COMMENTED** [review]
Some small comments :)

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc (hunk 1 above)
Comment (rolfedh) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc]: Updated. Thank  you. Made sure there weren't any similar instances to fix in this file.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc [hunk 3]
```diff
@@ -286,12 +286,12 @@ The Red Hat Container Catalog no longer exists; it has become part of the Red Ha
 // EAP: General; kept as is
 [[red-hat-customer-portal]]
 ==== image:images/yes.png[yes] Red Hat Customer Portal (noun)
-*Description*: _Red Hat Customer Portal_ is the official name of the customer portal at https://access.redhat.com.
+*Description*: _Red Hat Customer Portal_ is the official name for https://access.redhat.com. Always use _Red Hat Customer Portal_ in the first reference. For subsequent mentions, you can shorten it to _Customer Portal_.
```
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:289]: We do quotes when directing them specific text to use

```suggestion
*Description*: _Red Hat Customer Portal_ is the official name for https://access.redhat.com. Always use "Red Hat Customer Portal" in the first reference. For subsequent mentions, you can shorten it to "Customer Portal".
```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc (hunk 2 above)
Comment (bergerhoffer) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc]: Per our discussion in the SSG meetings, we decided against using just "the portal". So I'd keep CP and RHCP, but I'd also keep "portal" here.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc [hunk 4]
```diff
@@ -286,12 +286,12 @@ The Red Hat Container Catalog no longer exists; it has become part of the Red Ha
...
 
 *Use it*: yes
 
 [.vale-ignore]
-*Incorrect forms*: Customer Portal
```
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:294]: Could you also add an entry for "Customer Portal"?

**Review by bergerhoffer — COMMENTED** [review]
Thanks for submitting this!

Two pieces of feedback here, then also commented on an earlier review to add "portal" to the list of incorrect forms.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc (hunk 3 above)
Comment (rolfedh) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc]: Thanks. Updated.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc (hunk 4 above)
Comment (rolfedh) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:294]: Added. Thanks. WDYT?

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc (hunk 2 above)
Comment (rolfedh) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc]: Based the SSG discussion, I've added "portal" back.

**Review by bergerhoffer — APPROVED** [review]
LGTM!

**Review by mportman12 — APPROVED** [review]
LGTM

**Review by bburt-rh — APPROVED** [review]
LGTM. Merging

**Pull request merged**